src/braket/circuits/gates.py

```diff
@@ -868,9 +1044,8 @@ def cnot(control: QubitSetInput, target: QubitInput) -> Instruction:
         """Registers this function into the circuit class.
 
         Args:
-            control (QubitSetInput): Control qubit(s). The last control qubit is absorbed
-                into the cnot gate. The result is equivalent to an x gate applied to
-                the target qubit and controlled by all control qubits.
+            control (Optional[QubitSetInput]): Control qubit(s). The last control qubit
+                is absorbed into the target of the instruction. Default None.
```

Comment:
What happens if control is `None`?
```suggestion
            control (QubitSetInput): Control qubit(s). The last control qubit
                is absorbed into the target of the instruction.
```